### PR TITLE
Fixing versions

### PR DESCRIPTION
**Pull request by raclepoulpe**
Fixing:
- helm command
- Xonotic version
- ingame pictures

### CODE REVIEW

**Review by LostInBrittany — APPROVED** [review]
That was FAST, thanks mate!

File: pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md [hunk 1]
```diff
@@ -105,10 +103,11 @@ Now let's continue by adding Agones repository to Helm's repository list.
 helm repo add agones https://agones.dev/chart/stable
 ```
 
-And then installing the stable Agones chart:
+And then, after creating the `agones-system` namespace, install the stable Agones chart:
 
 ```bash
-helm install --name my-agones --namespace agones-system agones/agones
+kubectl create namespace agones-system
+helm install my-agones --namespace agones-system agones/agones
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md]: ou bien `helm install my-agones --namespace agones-system agones/agones --create-namespace` pour eviter la commande de creation du namespace avant

**Review by scraly — COMMENTED** [review]
Top, thanks!

Just one remark about the usage of only one command to install with helm CLI and create namespace if not exists.

File: pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md (hunk 1 above)
Comment (raclepoulpe) [reply pages/platform/kubernetes-k8s/installing-agones/guide.en-asia.md]: Merci :)

**Comment by OVH-Team-Guides** [comment]
Thank you for you contribution @raclepoulpe

**Pull request merged**